addons/web/static/src/views/calendar/calendar_renderer.scss

```diff
@@ -0,0 +1,697 @@
+.o_calendar_view {
```

Comment:
you sure about this? this isn't the renderer node, so either the classname isn't correct, or these rules aren't in the correct file